source/cellular/known-working-3g-4g-modems.rst

```diff
@@ -53,11 +53,6 @@ pfSense 2.0
    -  Link interface: /dev/cuaU0.1
    -  Init string: &F&C1&D2E0S0=0
 
--  Huawei E3372h LTE USB-stick
-
-   -  Requires manual firmware changes, see
-      `here <http://www.0xf8.org/2017/01/flashing-a-huawei-e3372h-4g-lte-stick-from-hilink-to-stick-mode/>`__
```

Comment:
Why remove this entirely? It's a viable workaround and it is a permanent fix which would not require configuring a command to run at every boot/insertion of the device.